tools/wptrunner/wptrunner/executors/testharness_webdriver.js

```diff
@@ -1,6 +1,17 @@
+var callback = arguments[arguments.length - 1];
```

Comment:
This is still run synchronously in Selenium, no?

Reply:
@jgraham I think this is the only thing blocking this landing?

Reply:
`executorselenium.py` runs this script synchronously, which means it won't wait for the window to navigate.

Reply:
See https://github.com/web-platform-tests/wpt/blob/fe7663150d5b90e1fe830b898701bd04a29ff216/tools/wptrunner/wptrunner/executors/executorselenium.py#L321 and contrast with `executorwebdriver.py`.

Reply:
Oh I see! Good catch.